msbuild/Xamarin.iOS.Tasks.Core/Tasks/MTouchTaskBase.cs

```diff
@@ -391,7 +391,7 @@ protected override string GenerateCommandLineCommands ()
 				args.Add ("--extension");
 
 			if (Debug) {
-				if (FastDev && IPhoneSdks.MonoTouch.SupportsFastDev)
+				if (FastDev)
```

Comment:
ok, but ideally this should have been a different PR, i.e. one with bump and API adjustment and the 2nd for the new task